src/Microsoft.AspNet.Mvc.ViewFeatures/Buffer/ViewBufferPage.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNet.Mvc.ViewFeatures.Buffer
+{
+    public class ViewBufferPage
+    {
+        public ViewBufferPage(ViewBufferValue[] buffer)
+        {
+            Buffer = buffer;
+        }
+
+        public void Append(ViewBufferValue value)
+        {
+            if (IsFull)
+            {
+                throw new InvalidOperationException();
```

Comment:
Removed this check, we'll just let the array access throw. This is going to be `.Internal` soon.